source/draw/Image.ooc

```diff
@@ -43,6 +43,8 @@ Image: abstract class {
 		this _canvas = null
 		super()
 	}
+	getFirstPacked: virtual func -> This { this }
```

Comment:
All images contains one or more packed images and Image is the only common parent of both YUV images.